### PR TITLE
[fix](compile) compile fail on MacOS because 'bits/types/struct_timespec.h' not found

### PR DESCRIPTION
**Pull request by Tech-Circle-48**
## Proposed changes

Issue Number: close #xxx

macos compilation in doris/be/src/runtime/fragment_mgr.cpp reports fatal error: 'bits/types/struct_timespec.h' file not found

### CODE REVIEW

File: be/src/http/action/stream_load.h [hunk 1]
```diff
@@ -1,3 +1,11 @@
+/*
+ * @Author: 宋光璠 songguangfan@sf.com
+ * @Date: 2024-08-19 10:33:33
+ * @LastEditors: 宋光璠 songguangfan@sf.com
+ * @LastEditTime: 2024-08-19 16:35:28
+ * @FilePath: /doris/be/src/http/action/stream_load.h
+ * @Description: 这是默认设置,请设置`customMade`, 打开koroFileHeader查看配置 进行设置: https://github.com/OBKoro1/koro1FileHeader/wiki/%E9%85%8D%E7%BD%AE
+ */
```
Comment (morrySnow) [inline be/src/http/action/stream_load.h:8]: remove this

Comment (Tech-Circle-48) [reply be/src/http/action/stream_load.h]: already remove

**Comment by zclllyybb** [comment]
run buildall

**Review by Yukang-Lian — APPROVED** [review]

**Review by morrySnow — APPROVED** [review]

**Comment by morrySnow** [comment]
run buildall

**Comment by morrySnow** [comment]
run buildall

**Comment by morrySnow** [comment]
run external

**Comment by zy-kkk** [comment]
run external

**Review by morrySnow — APPROVED** [review]

**Pull request merged**